src/System.Security.Cryptography.X509Certificates/tests/project.json

```diff
@@ -23,13 +24,10 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00520-02"
   },
   "frameworks": {
-    "netstandard1.4": {}
+    "netstandard1.6": {}
   },
   "supports": {
-    "coreFx.Test.netcore50": {},
```

Comment:
OK. I'll look at this. I removed this when I was having issues trying to get the correct S.S.C.OpenSsl version. Wasn't sure we still run the netcore50 tests